src/main/java/com/chaintope/openassetsj/utils/Utils.java

```diff
@@ -55,16 +58,11 @@ public static String addressToOaAddress(Address address) {
         
```

Comment:
azuchi san say follow on the slack:
> azuchi [15days ago]
> It is not necessary to have the same as openassets-ruby.
> For example, can not make Open Assets Address extends from org.bitcoinj.core.VersionedChecksummedBytes > or org.bitcoinj.core.Address instead of implementing it in Util?

so, you should create OAAddress class, and then move thease source to OAAddress class. The Util class is evil.....